cluster-autoscaler/simulator/drain.go

```diff
@@ -130,7 +134,7 @@ func checkPdbs(pods []*apiv1.Pod, pdbs []*policyv1.PodDisruptionBudget) (*drain.
 
 func drainabilityStatus(pod *apiv1.Pod, dr []drainability.Rule) drainability.Status {
```

Comment:
I think this function would be more readable if as a default you return the `drainability.Status{Outcome: drainability.UndefinedOutcome}` instead of just `drainability.Status{}`. They're equivalent now, but this will both make this more readable and ensure that we won't break if someone accidentally changes the 0 value of `OutcomeType`

Reply:
`UndefinedOutcome` is a zero value by design, but I guess a little extra verbosity won't hurt. Updated.

Reply:
Thanks!